packages/project-access/src/file/file-search.ts

```diff
@@ -0,0 +1,91 @@
+import type { Editor } from 'mem-fs-editor';
+import { basename, dirname, join, sep } from 'path';
+import { default as find } from 'findit2';
+import { fileExists } from './file-access';
+
+/**
+ * Add missing dump properties
+ */
+declare module 'mem-fs-editor' {
+    type FileMap = { [key: string]: { state: 'modified' | 'deleted' } };
+
+    export interface Editor {
+        dump(cwd: string): FileMap;
```

Comment:
JS Doc for `dump` may be helpful

Reply:
Copied docu from https://github.com/SBoudrias/mem-fs-editor#dumpcwd-filter